lust/lib/widgets/getTextBox.dart

```diff
@@ -0,0 +1,35 @@
+import 'package:flutter/cupertino.dart';
+import 'package:flutter/material.dart';
+
+class TextBox {
+  //TextBox(this._text);
+  //final String _text;
+  //String inputUser;
+
+  static Widget getTextBox(BuildContext context, String _title, String _hint) {
+    String inputUser;
+
+    return new Row(
+      mainAxisAlignment: MainAxisAlignment.spaceBetween,
+      children: <Widget>[
+        Text(
+          _title,
+          style: TextStyle(fontSize: 20),
+        ),
+        TextField(
```

Comment:
wrap an expanded around the textfield
like so: https://stackoverflow.com/questions/54269313/fill-row-with-textfield